### DISCUSSION TITLE
怎么更改PDF阅读器的字体

### DISCUSSION THREAD

**Discussion by 410676306** [chunibyocola/sc-translator-crx#177] [Q&A]
这个是原字体：
![image](https://github.com/user-attachments/assets/b4750eaf-cefa-4e40-af49-25fd690e04b5)
这个是用浏览器打开后的字体（看得有点吃力）：
![image](https://github.com/user-attachments/assets/32248404-09cf-4477-bc96-dee337146e92)

**Comment by chunibyocola** [comment]
字体由扩展使用的第三方 PDF Viewer 控制，目前没有太好的方法去改变字体
